### DISCUSSION TITLE
[Question] Bimanual cartesian impedance control for Learnign from Demonstration

### DISCUSSION THREAD

**Discussion by mikelasa** [isaac-sim/IsaacLab#511] [Q&A]
### Question

Hello,

I would like to start in a bimanual cartesian impedance control, to further do learning from demonstration with both robots. I have seen in the isaac orbit documentation that Learning from Demonstration is available to simulate, but among all the examples I have seen, they are for one robot. I have seen as well that the motion generator that are available below include a variable impedance control which I'm very interested to use. Where can I find the implementation of the controller?
[https://isaac-orbit.github.io/orbit/source/features/motion_generators.html#task-space-controllers](url)

Is possible to control two robots in the same environment using two controllers? I'm quite new with the Nvidia isaac sim and orbit so maybe its a naive question.

**Comment by songwookim** [comment]
did you have anything progressed?
